model_analyzer/config/generate/quick_plus_concurrency_sweep_run_config_generator.py

```diff
@@ -59,8 +60,10 @@ def __init__(self, search_config: SearchConfig,
         config: ConfigCommandProfile
             Profile configuration information
         gpus: List of GPUDevices
-        models: List of ConfigModelProfileSpec
+        models: List of ModelProfileSpec
             List of models to profile
+        ensemble_submodels: Dict of List of ModelProfileSpec
```

Comment:
It looks like this could in theory support multi-model of any combination of ensemble and non-ensemble models?

Reply:
Yes, I believe that will work. But, at this point I'm not focusing on testing for those combinations.